src/app-layer-nbss.h

```diff
@@ -24,11 +24,6 @@
 #ifndef __APP_LAYER_NBSS_H__
```

Comment:
looks like this file can be removed instead? Should have been removed as part of the smb to rust work years ago I think

Reply:
Oh and so are decode-null.h and util-vector.h :-)